src/js/components/Select/__tests__/__snapshots__/Select-test.js.snap

```diff
@@ -2040,6 +2040,11 @@ exports[`Select applies custom global.hover theme to options 2`] = `
   color: #000000;
 }
 
+.c0:hover {
```

Comment:
I believe this is a fix to behavior that previously should've been in the snapshot.